Report/sections/2_design/Design_evaluation.tex

```diff
@@ -0,0 +1,46 @@
+\section{Design evaluation} \label{chap:design evaluation}
+What is significant about developing a language is to carefully consider various parameters that could be around the language itself and the setting up of essential criteria for the programming language. 
+Therefore, it is crucial to be able to evaluate the different parameters using different tools. Thus, the group has chosen to use an already set up table, which covers all the most important criteria for setting up a programming language. 
+It has been selected as the need to find other parameters seemed unnecessary, as this table contains all the critical parameters behind the coding language to be produced.
+
+
+
+\fig{0.8}{Design_Criteria}{Table about design criteria }
+
+
+
+The simplicity is to be able to keep the language simple to read and understand for everyone. It helps to keep the learning curve at a superficial level.
+
+Orthogonality is about how many different ways to put together data structures. High orthogonality describes that the data structure of the language can be composed in many different ways.
+
+Data types are the different types of data that are possible to work within the language. By simplifying the data type, names in the language can help the beginner understand the various properties behind the language.
+
+Syntax design is a broad concept within the language, which can be the actual use of unique and reserved words.
+Abstraction support is the precise handling of the code's reuse in the language itself without repeating the code writing.
+
+Expressiveness is the very meaning of ways one can write down different operations in the language. Being able to make them simple to write down helps make the language and readability better.
+
+Type checks are used to be able to test type errors that could appear in the program. This can be done by either the compiler intercepts it or when the program is running. There can be many different type checks in a programming language. It helps that you quickly have a chance to detect faults in good time and solve them quickly.
```

Comment:
```suggestion
Type checks are used to be able to test type errors that could appear in the program. This can be done by either the the compiler intercepting it or when the program is running. There can be many different type checks in a programming language. It helps detecting faults in good time and solve them quickly.
```